akd/src/storage/transaction.rs

```diff
@@ -40,6 +40,12 @@ impl std::fmt::Debug for Transaction {
     }
 }
 
+impl Default for Transaction {
+    fn default() -> Self {
+        Self::new()
+    }
+}
+
```

Comment:
It looks like we can add `#[derive(Default)]` to `TransactionState` and `Transaction` if we'd like to avoid the the explicit impl here.